libraries/protocol/steem_operations.cpp

```diff
@@ -598,25 +598,30 @@ namespace golos { namespace protocol {
         }
 
         void decline_voting_rights_operation::validate() const {
-            validate_account_name(account);
+            GOLOS_CHECK_PARAM_ACCOUNT(account);
         }
 
         void reset_account_operation::validate() const {
-            validate_account_name(reset_account);
-            validate_account_name(account_to_reset);
-            FC_ASSERT(!new_owner_authority.is_impossible(), "new owner authority cannot be impossible");
-            FC_ASSERT(new_owner_authority.weight_threshold, "new owner authority cannot be trivial");
-            new_owner_authority.validate();
+            // This op is disabled. Maybe just remove it completely?
+            GOLOS_CHECK_PARAM_ACCOUNT(reset_account);
+            GOLOS_CHECK_PARAM_ACCOUNT(account_to_reset);
+            GOLOS_CHECK_PARAM(new_owner_authority, {
+                GOLOS_CHECK_VALUE(!new_owner_authority.is_impossible(), "New owner authority cannot be impossible");
+                GOLOS_CHECK_VALUE(new_owner_authority.weight_threshold, "New owner authority cannot be trivial");
+                new_owner_authority.validate();
```

Comment:
What about validation in `authority::validate()`?

Reply:
It already updated with custom_ops commit